madoop/mapreduce.py

```diff
@@ -202,7 +202,7 @@ def map_stage(exe, input_dir, output_dir):
                     f"Command returned non-zero: "
                     f"{exe} < {input_path} > {output_path}"
                 ) from err
-    LOGGER.info("Finished map executions: %s", i)
+    LOGGER.info("Finished map executions: %s", i+1)
```

Comment:
ohhh good catch.  I nitpicked the solution ac22df3e124124f44405eb5b434be6052e28d91b